bool get_Mass          # Flag to request Mass matrix
bool get_Coriolis      # Flag to request Coriolis matrix
bool get_Gravity       # Flag to request Gravity vector
---
float32[9] Mass       # 3x3 mass matrix, row-major
float32[9] Coriolis   # 3x3 coriolis matrix, row-major
float32[3] Gravity    # 3x1 gravity vector